dotlottie-rs/tests/theming.rs

```diff
@@ -79,4 +87,81 @@ mod tests {
 
         assert!(player.load_theme(""), "Expected theme to unload");
     }
+
+    #[test]
+    fn test_clear_active_theme_id_after_new_animation_data_is_loaded() {
+        let player = DotLottiePlayer::new(Config {
+            autoplay: true,
+            ..Config::default()
+        });
+
+        let valid_theme_id = "test_theme";
+
+        assert!(
+            !player.load_theme(valid_theme_id),
+            "Expected theme to not load"
+        );
+
+        assert!(player.load_dotlottie_data(include_bytes!("assets/test.lottie"), WIDTH, HEIGHT));
+
+        assert!(player.load_theme(valid_theme_id), "Expected theme to load");
+        assert_eq!(player.active_theme_id(), valid_theme_id);
+
+        let mut test_json_file = File::open("tests/assets/test.json").expect("no file found");
+        let metadata = fs::metadata("tests/assets/test.json").expect("unable to read metadata");
+        let mut buffer = vec![0; metadata.len() as usize];
+        test_json_file.read(&mut buffer).expect("buffer overflow");
+        let string = String::from_utf8(buffer.clone()).unwrap();
```

Comment:
```suggestion
        let data =
        std::str::from_utf8(include_bytes!("assets/test.json")).expect("Invalid data.");
```

Reply:
Use the `include_bytes!` macro to embed the animation file data directly into the binary at compile time. This method is somewhat faster than opening the file, reading metadata, dynamically allocating a buffer in memory, as it avoids some runtime overhead.

Reply:
updated